vaas-app/src/vaas/manager/api.py

```diff
@@ -69,6 +70,33 @@ class Meta:
             'cluster': ALL_WITH_RELATIONS
         }
 
+    def save_m2m(self, bundle):
+        logger = logging.getLogger('vaas')
+        try:
+            new_uris = bundle.obj.new_clusters_uris
+            bundle.obj.new_clusters = [cluster.obj for cluster in bundle.data['cluster']
+                                       if cluster.data['resource_uri'] in new_uris]
+            logger.info("[DirectorResource.save_m2m()] new_clusters = {}".format(bundle.obj.new_clusters))
```

Comment:
don't use `.format()` for logger messages, use standard % formatting so string interpolation happens in the logging code https://github.com/openstack-dev/hacking/blob/master/hacking/checks/other.py#L39